main/src/main.rs

```diff
@@ -77,34 +76,21 @@ struct RunArgs {
     #[arg(long, global = true)]
     config: Option<String>,
 
-    /// The deployment mode of CnosDB.
-    /// There are Tskv, Query, QueryTskv and Singleton, the default is QueryTskv
+    /// The deployment mode of CnosDB,
+    ///  default value is query_tskv
     #[arg(short = 'M', long, global = true, value_enum)]
     deployment_mode: Option<DeploymentMode>,
 }
 
-#[derive(Debug, Copy, Clone)]
+#[derive(Debug, Copy, Clone, ValueEnum)]
+#[clap(rename_all = "snake_case")]
 enum DeploymentMode {
     Tskv,
```

Comment:
Here we need some comments to show help messages as below:

```rust
#[derive(Parser)]
#[command(author, version, about, long_about = None)]
struct Cli {
    /// What mode to run the program in
    #[arg(value_enum)]
    mode: Mode,
}

#[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, ValueEnum)]
enum Mode {
    /// Run swiftly
    Fast,
    /// Crawl slowly but steadily
    ///
    /// This paragraph is ignored because there is no long help text for possible values.
    Slow,
}
```

```sh
......
Arguments:
  <MODE>
          What mode to run the program in

          Possible values:
          - fast: Run swiftly
          - slow: Crawl slowly but steadily
......
```